spring-cloud-gcp-samples/spring-cloud-gcp-security-iap-sample/README.adoc

```diff
@@ -0,0 +1,7 @@
+TODO
+
+Note: this sample should be tested when deployed to AppEngine Flexible or Kubernetes environments.
+
+It is possible, in principle, to grab a recent JWK token from a deployed apps' /headers path, and to test locally, but this token will expire, causing authentication to be declined locally.
+
+If you don't create your own implementation of WebSecurityConfigurerAdapter, Spring Boot will introduce its own, OAuth2ResourceServerWebSecurityConfiguration
```

Comment:
The classes are definitely code. so, please, proper wrapping.
Also every sentence should finish with period.